image/tools/lib/component/codeready_pv.sh

```diff
@@ -8,17 +8,18 @@ function dump_pod_data {
 }
 
 function component_dump_data {
-    workspace_pods=$(oc get pods -n ${PRODUCT_NAMESPACE_PREFIX}codeready | grep workspace | awk '{print $1}')
-
-    if [ "${#workspace_pods}" -eq "0" ]; then
-        echo "No workspaces found to backup"
-    else
-        archive_path="$1/archives"
-        dump_dest="/tmp/codeready-data"
-        mkdir -p $dump_dest
-        for i in $workspace_pods; do dump_pod_data $i $dump_dest; done
-        local ts=$(date '+%H_%M_%S')
-        tar -zcvf "$archive_path/codeready-pv-data-${ts}.tar.gz" -C $dump_dest .
-        rm -rf $dump_dest
+    local pods="$(oc get pods -n ${PRODUCT_NAMESPACE_PREFIX}codeready | grep workspace)"
+    if [ "${#pods}" -eq "0" ]; then
+        echo "=>> No workspaces found to backup"
+        exit 0
```

Comment:
couldnt find a better way to handle this in the entrypoint file so I decided to exit here